src/main/java/org/elasticsearch/kafka/indexer/jobs/ConsumerWorker.java

```diff
@@ -146,10 +157,14 @@ public void run() {
      * if you want to expose custom JMX metrics
      * @param previousPollEndPosition
      */
-    protected void exposeOffsetPositionToJmx(Map<TopicPartition, OffsetAndMetadata> previousPollEndPosition) {
+    public void exposeOffsetPositionToJmx(Map<TopicPartition, OffsetAndMetadata> previousPollEndPosition) {
```

Comment:
it can be more options than only jmx, I'd name it just `exposeOffsetPosition`

Reply:
done